### PR TITLE
Remove views guides

### LINKED ISSUE #3933: According to user feedback, our database views guide breaks their ability to do migrations
A user says that our [guide on views](https://www.prisma.io/docs/guides/database/advanced-database-tasks/sql-views-postgres) breaks users' ability to do migrations if they follow it. This is something we should investigate and, if it's broken, fix or remove as a priority. [User feebdack]( https://github.com/prisma/prisma/issues/678#issuecomment-1284623234)

[Jan comments](https://prisma-company.slack.com/archives/C5Z9TH6N9/p1666302569285729) that this and the other two advanced database tasks send migrate users onto a bad path, so we should also investigate and potentially fix or remove the other two guides in this group of topics.

### PR DESCRIPTION
**Pull request by keerlu**
## Describe this PR

Remove views guide and cascading deletes guides and all references to them

## What issue does this fix?

Fixes #3933

### CODE REVIEW

File: content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx [hunk 1]
```diff
@@ -181,17 +181,6 @@ const createTeamOperation = prisma.team.create({
...
-##### Are cascading deletes supported by the Prisma Client nested writes?
-
-<Admonition type="warning">
-
-In [2.26.0](https://github.com/prisma/prisma/releases/tag/2.26.0) and later it is possible to do cascading deletes using [referential actions](/concepts/components/prisma-schema/relations/referential-actions).
-
-</Admonition>
-
-- Use the `$transaction` API to perform a cascading delete. Deleting a team and deleting the team's members are not dependent writes - if you know the ID of the team, you can [perform both operations within a `$transaction`](#scenario-privacy-legislation).
-- [Configure cascading deletes at the database level](/guides/database/advanced-database-tasks/cascading-deletes).
```
Comment (keerlu) [inline content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx:193]: Can this section just be deleted or do we need a version of it around still?

Comment (janpio) [reply content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx:193]: Investigation: https://prisma-company.slack.com/archives/CC59J6VEY/p1667508837505919

File: vercel.json [hunk 2]
```diff
@@ -657,27 +653,11 @@
       "source": "/guides/deployment/deploying-to-netlify",
       "destination": "/docs/guides/deployment/deployment-guides/deploying-to-netlify"
     },
-
-    {
-      "source": "/guides/general-guides/database-workflows/cascading-deletes/:any*",
-      "destination": "/docs/guides/database/advanced-database-tasks/cascading-deletes/:any*"
```
Comment (janpio) [inline vercel.json]: These two URLs should probably redirect to our Referential Actions page now instead.

File: vercel.json [hunk 3]
```diff
@@ -657,27 +653,11 @@
...
 
-    {
-      "source": "/guides/general-guides/database-workflows/sql-views-postgres",
-      "destination": "/docs/guides/database/advanced-database-tasks/sql-views-postgres"
-    },
-
-    {
-      "source": "/guides/general-guides/database-workflows/sql-views-mysql",
-      "destination": "/docs/guides/database/advanced-database-tasks/sql-views-mysql"
-    },
```
Comment (janpio) [inline vercel.json]: Maybe redirect all the Views URLs to the GitHub issue instead?

**Review by janpio — APPROVED** [review]

File: vercel.json (hunk 2 above)
Comment (keerlu) [reply vercel.json]: Added in these redirects

File: vercel.json (hunk 3 above)
Comment (keerlu) [reply vercel.json]: Added in these redirects

File: content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx (hunk 1 above)
Comment (janpio) [reply content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx:193]: Could not get definitive information, but probably not that relevant any more. People should upgrade.

**Pull request merged**